src/SbManager/Content/tmpl/directives/error.html

```diff
@@ -0,0 +1,4 @@
+﻿<div class="panel panel-danger">
+    <div class="panel-heading">{{model.Error.Title || "Sorry, something went wrong"}}</div>
+    <div class="panel-body">{{model.Error.Summary || "An unexpected error occurred."}}</div>
```

Comment:
Be nice if we didn't loose context of the error message that would have been displayed on the alert

```